dashboard/settings.py

```diff
@@ -484,6 +486,12 @@
         'django.contrib.auth.backends.ModelBackend',
     )
 
+CONSTANCE_BACKEND = 'constance.backends.database.DatabaseBackend'
+
+CONSTANCE_CONFIG = {
+    'SURVEY_URL': ('', 'URL To qualtrics survey', str),
+}
```

Comment:
Yeah, I wanted to get this in first then we could figure out the other configs. I could have made adding CONSTANCE a separate issue but it's really simple and not really worthwhile without adding *something*

Reply:
I created #1200 to capture this request for a future release.

Reply:
Cool, to be clear, I'm not objecting to the shoehorning of `constance` here, I just wanted to make sure we document what could be done in the future with this. It could be related to some of the other configuration issues we have open.